src/styles/_mixins.scss

```diff
@@ -22,58 +22,58 @@
 }
 
 @mixin input-default {
-  background-color: var(--KInputBackground, var(--white, color(white)));
-  box-shadow: inset 0 0 0 1px var(--KInputBorder, var(--grey-300)) !important;
+  background-color: var(--KInputBackground, var(--white, #ffffff));
+  box-shadow: inset 0 0 0 1px var(--KInputBorder, var(--grey-300, #e7e7ec)) !important;
   transition: color 0.1s ease, box-shadow 0.1s ease;
 }
 
 @mixin input-focus {
-  box-shadow: inset 0 0 0 1px var(--KInputFocus, var(--blue-400)) !important;
+  box-shadow: inset 0 0 0 1px var(--KInputFocus, var(--blue-400, #3972d5)) !important;
   outline: none !important;
   transition: all 0.1s ease;
 }
 
 @mixin textarea-default {
-  background-color: var(--KInputBackground, var(--white, color(white)));
+  background-color: var(--KInputBackground, var(--white, #ffffff));
```

Comment:
question: can we use $kui-color-background here?

Reply:
See above